packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1514,8 +1524,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             }).map(async ([key, value]) => {
                 const contextSummary = await value.summarize(fullTree, trackState);
                 builder.addWithStats(key, contextSummary);
+                nodes.push(...contextSummary.nodes);
```

Comment:
It might be useful to validate inputs, i.e. that data store nodes all have same prefix in their paths (starting with data store ID).

Reply:
I will handle this in a seprate PR.